common/src/main/java/com/sequenceiq/cloudbreak/telemetry/TelemetryUpgradeConfiguration.java

```diff
@@ -0,0 +1,42 @@
+package com.sequenceiq.cloudbreak.telemetry;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TelemetryUpgradeConfiguration {
```

Comment:
you can use `@ConfigurationProperties` here too to avoid defining config key for each value